### PR TITLE
[JAV-548] Pojo consumer upload

### PR DESCRIPTION
**Pull request by wujimin**

### CODE REVIEW

File: foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestAbstractPart.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
+/*
```
Comment (WillemJiang) [inline foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestAbstractPart.java]: Please update the license header with the new one.

File: foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestFilePart.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
+/*
```
Comment (WillemJiang) [inline foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestFilePart.java]: Please update the license header with the new one.

File: foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestInputStreamPart.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
+/*
```
Comment (WillemJiang) [inline foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestInputStreamPart.java]: Please update the license header with the new one.

File: swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/extend/property/creator/PartPropertyCreator.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
+/*
```
Comment (WillemJiang) [inline swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/extend/property/creator/PartPropertyCreator.java]: Please update the license header with the new one.

File: swagger/swagger-generator/generator-core/src/test/java/io/servicecomb/swagger/extend/property/creator/TestPartPropertyCreator.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
+/*
```
Comment (WillemJiang) [inline swagger/swagger-generator/generator-core/src/test/java/io/servicecomb/swagger/extend/property/creator/TestPartPropertyCreator.java]: Please update the license header with the new one.

**Comment by liubao68** [comment]
Please resolve conflicts and then I'll aprove

File: foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestAbstractPart.java (hunk 1 above)
Comment (wujimin) [reply foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestAbstractPart.java]: done

File: foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestFilePart.java (hunk 2 above)
Comment (wujimin) [reply foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestFilePart.java]: done

File: foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestInputStreamPart.java (hunk 3 above)
Comment (wujimin) [reply foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestInputStreamPart.java]: done

File: swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/extend/property/creator/PartPropertyCreator.java (hunk 4 above)
Comment (wujimin) [reply swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/extend/property/creator/PartPropertyCreator.java]: done

File: swagger/swagger-generator/generator-core/src/test/java/io/servicecomb/swagger/extend/property/creator/TestPartPropertyCreator.java (hunk 5 above)
Comment (wujimin) [reply swagger/swagger-generator/generator-core/src/test/java/io/servicecomb/swagger/extend/property/creator/TestPartPropertyCreator.java]: done

**Pull request merged**